C/CUDA/CUDA_full@10.2/build_tarballs.jl

```diff
@@ -18,7 +18,55 @@ sources_windows = [
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/c/cuda-nvrtc/cuda-nvrtc-10-2_10.2.300-1_arm64.deb", "7118079394a22c6cedba1152f0a78ef7bb10ac26c1984ab8c4dcf82fe9f4e20c"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/c/cuda-nvrtc/cuda-nvrtc-dev-10-2_10.2.300-1_arm64.deb", "bdce95a7a30daf7d1a52a32385c1ee67a8f742126a2f24d82a7b7dd35d012f3f"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcusolver/libcusolver-10-2_10.3.0.300-1_arm64.deb", "31a32e7c7e439bc34331d32b93392d08de6c9893d5083865b93703509d6fb7d6"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcusolver/libcusolver-dev-10-2_10.3.0.300-1_arm64.deb", "c86b549b4501a8ddd451dd7f46ea55b79d548bd987aaa17f3af03813e4b90dba"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcublas/libcublas10_10.2.3.300-1_arm64.deb", "a4cde28215c01bce78fa7c3d09909afaec487142e1144646bd7ff043a5c6c1df"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcublas/libcublas-dev_10.2.3.300-1_arm64.deb", "f2575fd6c86c9aa8eed0e782d68122d9ba3d1422ebd6e6c06c71bea096c169db"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcufft/libcufft-10-2_10.1.2.300-1_arm64.deb", "661a91f35f2d148d938ca907484ebe48727b17f03a8cbb6992e60bb792738caf"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcufft/libcufft-dev-10-2_10.1.2.300-1_arm64.deb", "43bf089682f745397c7c72bb53ce6d962e2d24246e1e7e686fd8159d3ddf7ba1"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcurand/libcurand-10-2_10.1.2.300-1_arm64.deb", "45f2bbafdde70b8b8bcdc6f0625a2ab334632b6659f153224654d03e1776057b"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcurand/libcurand-dev-10-2_10.1.2.300-1_arm64.deb", "7da1321d221290c7c8dcf49bef8970712cb981d8f9a93f405a080364f1bf0b12"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcusparse/libcusparse-10-2_10.3.1.300-1_arm64.deb", "bd844b7a71adc76694caf5da4818ab03f207e3b7f23340aa2fe86909f799fb92"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libc/libcusparse/libcusparse-dev-10-2_10.3.1.300-1_arm64.deb", "898fab6cae719007fdf0c8ebb27825bc7dd03e48314297b2b60639b604fa6bc4"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libn/libnpp/libnpp-10-2_10.2.1.300-1_arm64.deb", "dc51dc218b7652680138ca236877d01ff0b65546d60a113276ded05f1fea967e"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/libn/libnpp/libnpp-dev-10-2_10.2.1.300-1_arm64.deb", "4c64992156eaeb982af504a41920a33df01c701286903d07fe23c1d20c902b38"),
+    FileSource("https://repo.download.nvidia.com/jetson/common/pool/main/c/cuda-samples/cuda-samples-10-2_10.2.300-1_arm64.deb", "bb7726194bac9252863da80ae13e4fdd7e69a657314cb8ff6edf8ba1cd789e2d"),
```

Comment:
I don't think the samples are part of regular CUDA distributions (at least not in the 10.2 Linux one I checked here), so we shouldn't install it here either.

Reply:
Right. Omitted samples.

And found the proper version (10.2.89) in the Jetson release 32.5.

Reply:
Why downgrade though? Is there a reason you want 10.2.89 and not .300?